src/WebJobs.Extensions.DurableTask/DefaultPlatformInformationProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class DefaultPlatformInformationProvider : IPlatformInformationService
```

Comment:
I forgot to add a header comment to this class. Will do later.
Essentially, this class looks up env-var values using the DI-provided nameResolver